packages/cli/medusa-cli/src/create-cli.ts

```diff
@@ -321,6 +321,19 @@ function buildLocalCommands(cli, isLocalProject) {
         })
       ),
     })
+    .command({
+      command: `script [file]`,
```

Comment:
How about `exec` or `execute`? `script` is not a verb so it reads a bit strangely.

Reply:
The idea was to expose a command with an immediately obvious purpose. That said, I imagine most are familiar with the concept of `exec`, so I am also fine with that. @shahednasser made a similar comment, so with 2/2 saying `script` is not super clear, I think it makes sense to reconsider the naming 😄 

cc @srindom